docs/install.md

```diff
@@ -46,21 +36,30 @@ secret is created and available before spawning the **aws-load-balancer-controll
     of secrets in a **manifests** directory that is required
     by the **aws-load-balancer-controller**.
 
+2. Use the `ccoctl` tool to create the controller IAM role and policy
+
+    ```bash
+    ccoctl aws create-iam-roles \
+        --name aws-load-balancer-controller-role --region=<aws_region> \
+        --credentials-requests-dir=./hack/controller \
+        --identity-provider-arn <oidc-arn>
+    ```
+
 3. Apply the secrets to your cluster:
 
     ```bash
-    ls manifests/*-credentials.yaml | xargs -I{} oc apply -f {}
+    find ./manifests/ -name "*credentials.yaml" | xargs -I{} oc apply -f {}
```

Comment:
What is the added value of this change? Find's wildcard is not different from bash's one.